library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
```

Comment:
What I meant about instance management is something like Spring which can easily configure the property of instances. As we don't have this mechanism, so the client code very difficult to get the instance(for this case is Bucket) to configure the property. That's why provide a static default value, client code can't easily change the default value. The unit test ManagedTimeStateImplTest.testAsyncGetFromReaders() is one example